ouroboros-network/src/Ouroboros/Network/Subscription/Worker.hs

```diff
@@ -155,7 +155,6 @@ sockAddrFamily
 sockAddrFamily (Socket.SockAddrInet  _ _    ) = Socket.AF_INET
 sockAddrFamily (Socket.SockAddrInet6 _ _ _ _) = Socket.AF_INET6
 sockAddrFamily (Socket.SockAddrUnix _       ) = Socket.AF_UNIX
-sockAddrFamily _                              = Socket.AF_UNSPEC
```

Comment:
I've updated to `network-3.0.0.0`. Now `SockAddr` has only these 3 variants.

Reply:
CI might fail (at least it failed for me when I needed to upgrade to `3.1`) in another PR. I will merge the fix soon.